### DISCUSSION TITLE
Multiple display support

### DISCUSSION THREAD

**Discussion by CaptChaos2** [rocksdanister/lively#1355] [General]
Love the live wallpaper product.  Problem (perhaps?) is the wallpaper only shows on primary display not both of them.
Cannot see a setting to adjust or is it me?

**Comment by rocksdanister** [comment]
<img width="647" alt="image" src="https://user-images.githubusercontent.com/17554161/185345585-705471a8-f608-45de-b66a-c91ce33bd9d8.png">

<img width="647" alt="image" src="https://user-images.githubusercontent.com/17554161/185345618-5b49b77e-57a9-4467-8d1f-b1a11f2b8904.png">

**Reply by CaptChaos2** [reply]
Thank you.  I am clearly as thick as a whale omlette!
